benchmarks/xla_flags_library.py

```diff
@@ -34,9 +34,7 @@
 
 DENSE_VMEM_LIMIT_FLAG = f" --xla_tpu_scoped_vmem_limit_kib={_DENSE_VMEM_LIMIT}"
 MOE_VMEM_LIMIT_FLAG = f" --xla_tpu_scoped_vmem_limit_kib={_MOE_VMEM_LIMIT}"
-CUSTOM_VMEM_LIMIT_FLAG = (
-    lambda vmem_limit: f"--xla_tpu_scoped_vmem_limit_kib={vmem_limit}"
-)
+CUSTOM_VMEM_LIMIT_FLAG = "--xla_tpu_scoped_vmem_limit_kib={}".format
```

Comment:
Lets revert this, as this isn't always equivalent

Reply:
Was resolving this lint https://pylint.pycqa.org/en/latest/user_guide/messages/convention/unnecessary-lambda-assignment.html whilst concurrently avoiding adding a trivial function with a docstring. Checked all usages in the codebase and it seems fine. I can replace usages with `xla_flags_library.CUSTOM_VMEM_LIMIT_FLAG(vmem_limit=122880)` and this with `{vmem_limit}".format` if you like…

Reply:
That sounds good, thank you! I believe you might also need to do .format(vmem_limit=vmem_limit_value)

Reply:
Sure thing: now takes `vmem_limit` keyword argument